Python/Product/PythonTools/PythonTools/Repl/BasePythonReplEvaluator.cs

```diff
@@ -977,7 +977,8 @@ public DisconnectedException(string message)
             }
 
             private void SendString(string text) {
-                byte[] bytes = System.Text.Encoding.UTF8.GetBytes(text);
+                Debug.Assert(!string.IsNullOrEmpty(text), "text should not be null or empty");

```

Comment:
Are we calling this SendString up above in 959-962 with an empty string only to assert here?